src/components/Dashboard/TodoList.vue

```diff
@@ -1330,7 +1330,7 @@ export default class TodoList extends Mixins(AllowableActionsMixin, DateMixin, E
         amalgamation?.offices ||
         amalgamation?.contactPoint ||
         amalgamation?.parties ||
-        amalgamation?.shareClasses
+        amalgamation?.shareStructure?.shareClasses
```

Comment:
As per the updated interface.

Reply:
Could you add a check for any amalgamating businesses?

Also, please fix my typos on line 1328 :)

Reply:
Thanks! Resolve if you want.